Dockerfile.localdev

```diff
@@ -0,0 +1,9 @@
+FROM registry.redhat.io/rhel8/go-toolset:1.17.7-13 AS builder
```

Comment:
why do we need this an `Dockerfile.webhook`

you should be logged into the openshift CI cluster per https://docs.ci.openshift.org/docs/how-tos/use-registries-in-build-farm/ so you can pull the CI builder image

Reply:
I aws not aware about this doc, I will remove the localdev Dockerfile then

Reply:
fixed